packages/nc-gui-v2/components/smartsheet/Toolbar.vue

```diff
@@ -5,8 +5,8 @@ const { isGrid, isForm } = useSmartsheetStoreOrThrow()
 </script>
 
 <template>
-  <div class="nc-table-toolbar w-full py-1 flex gap-1 items-center" style="z-index: 7">
-    <SmartsheetToolbarSearchData v-if="isGrid" class="flex-shrink" />
+  <div class="nc-table-toolbar w-full py-1 flex gap-1 items-center h-[48px] px-2" style="z-index: 7">
+    <SmartsheetToolbarSearchData v-if="isGrid" class="flex-shrink mr-2" />
```

Comment:
replace flex-shrink with shrink or just use flex-auto, which should be fine as well.